mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -32,6 +32,9 @@ pipeline {
             steps {
                 checkout scm
                 sh '''
+                    cd ..
+                    mkdir $BUILD_TAG
+                    cd $BUILD_TAG
```

Comment:
It looks like BUILD_TAG is `jenkins-${JOB_NAME}-${BUILD_NUMBER}`. It is not clear to me why you want to create such a directory in MLIR's build directory and clone MIOpen in it.

Now  instead of 
```bash
cd ..
mkdir $BUILD_TAG
cd $BUILD_TAG
```

Do a 
```bash
cd ~
```